### DISCUSSION TITLE
showing repo status in prompt, i.e., zsh Powerlevel10k

### DISCUSSION THREAD

**Discussion by reagle** [jj-vcs/jj#4036] [Q&A]
I use the zsh Powerlevel10k prompt, which can show me my git status when in a git repo. Is there something similar for jj?

**Comment by johanneskastl** [comment]
In my understanding as a jj newbie, as soon as you execute a `jj` command, you might add all changes to the current change. So, unlike with git, running commands in your prompt is not non-intrusive. There are some command line arguments in discussion, but as far as I found there is no final solution yet.

**Reply by shanesveller** [reply]
`--ignore-working-copy` is a global flag that has been available since well before I began using `jj` and should prevent unintended contributions to repo history from read-only commands.

https://jj-vcs.github.io/jj/latest/cli-reference/#jj Under "Options"

**Reply by johanneskastl** [reply]
I thought I had seem some remark that `--ignore-working-copy` might not be completely sufficient to prevent changes. Hence my comment on the discussions.

I do no longer find the exact issue or discussion, unfortunately. So I daresay you should rather believe @shanesveller  than me :-)
